ui/components/imageWorkspace/src/ImageWorkspace.svelte

```diff
@@ -104,6 +109,30 @@
       onSave().catch((err) => console.error(err));
     }
   }
+
+  onMount(() => {
+    //TMP: should be removed when available value written in db.json, so present in DatasetInfo[] from getDatasets (S)
+    // but right now, we compute available feats only in getDataset (no S) route, used only for stats, and here
+    // later, we should get itemFeaturesAvailableValues directly from DatasetInfo["available_feat_values"]
+    api
+      .getDataset(currentDatasetId)
+      .then((datasetWithFeats) => {
```

Comment:
I feel like this is not located in the most accurate component. Data fetching is mainly done in `src/routes/layout.svelte` . Maybe that could be something to reconsider but for now I think we should try to locate as many connexion to the API with in there, what do you think?

Reply:
Done